R/utils.R

```diff
@@ -76,6 +76,12 @@ drop_named_nulls <- function(x) {
   cleanse_names(x[!named | !null])
 }
 
+.parse_params <- function(..., .params = list()) {
+  params <- c(list(...), .params)
+  params <- drop_named_nulls(params)
+  return(params)
```

Comment:
nitpick: Unnecessary `return()`

```suggestion
  drop_named_nulls(params)
```